maintainers/maintainer-list.nix

```diff
@@ -15484,4 +15484,10 @@
     github = "quasigod-io";
     githubId = 62124625;
   };
+  brendanreis = {
```

Comment:
This list is sorted alphabetically.

Reply:
Whoops, didn't notice because the last few people were not inserted in the correct spots, but I should have known not to just look at the end. Updated.